homeassistant/components/mobile_app/.translations/en.json

```diff
@@ -0,0 +1,15 @@
+{
+    "config": {
+        "abort": {
+            "single_instance_allowed": "Only a single configuration of Mobile App is necessary.",
+            "install_app": "Open the mobile app to set up the integration with Home Assistant. See our website for a list of compatible apps."
```

Comment:
We should make it a link to our website

Example on how to do that:
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/owntracks/strings.json#L14

https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/owntracks/config_flow.py#L55-L58

I see now that abort doesn't have `description_placeholders` yet, but should be easy to add here: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/data_entry_flow.py#L169-L173